cms_form/static/src/js/ajax.js

```diff
@@ -0,0 +1,80 @@
+odoo.define('cms_form.ajax', function (require) {
+    'use strict';
+
+    var core = require('web.core'),
+        animation = require('website.content.snippets.animation');
+
+    animation.registry.CMSFormAjax = animation.Class.extend({
+        selector: ".cms_form_wrapper form[data-ajax]",
+        start: function () {
+            var data = this.$el.data('form');
+            this.$el.on('submit', this.proxy('submit_form'));
+            if (data.ajax_onchange) {
+                this.$el.on('change', this.proxy('submit_form'));
```

Comment:
any reason not to use `events` dict?

Reply:
I'm not aware of a way to catch events conditionally on widgets, is there one by now save for selectors? It might make sense anyways to add this option in a data attribute and not in the json data

Reply:
`'change form[data-ajax-onchange]': 'submit_form'` this should do the trick

Reply:
for this i need to move the flag from the json dict to the form itself, is that what you want? (from the code I figured you wanted to avoid having many data attributes on the form)

Reply:
the events dict only looks at *children* as I noticed now, not the element itself. And the animation runs on the form, because that's where the data attribute is. I could change this but then form access is less convenient, so I think looking at the parent as I now do should be fine